scripts/copy_form.py

```diff
@@ -0,0 +1,271 @@
+import datetime as dt
+import json
+import uuid
+import boto3
+import argparse
+
+def scan(client, tablename, filterexpression, filterattributes=None, filtervalues=None):
+    items = []
+    res = {}
+    keywordargs = {}
+    if not filterattributes == None:
```

Comment:
Hva med `if filterattributes != None` i stedet, eller om du er på jakt etter å lese det som en setning: `if filterattributes is not None`